### DISCUSSION TITLE
给作者大大几条小建议

### DISCUSSION THREAD

**Discussion by long1512448773** [xiaye13579/BBLL#388] [General]
1:可以增加一个账号切换功能 因为是TV端 我妹也想看
2:优化一下视频 有些视频黑屏无法播
3:可以增加观看UP主粉丝功能吗？
4:能不能 多一些解码！因为有些缓存比较慢，不是网络问题

**Comment by long1512448773** [comment]
> 1:可以增加一个账号切换功能 因为是TV端 我妹也想看
> 2:优化一下视频 有些视频黑屏无法播
> 3:可以增加观看UP主粉丝功能吗？
> 4:能不能 多一些解码！因为有些缓存比较慢，不是网络问题
> 

还有关注的人显示不全 我是一个经常看哔哩哔哩TV的人 还有默认能不能 1080p呢 直播清晰度有待加强

**Reply by maoscs** [reply]
还有搜索的时候输入了搜索内容，但没办法确定。导致无法搜索
